### PR TITLE
Redirect "Learning environment" to Install Tools task

### LINKED ISSUE #15969: Examine Minikube pages for dual-hosted content and content that should move to minikube docs site
**This is a Feature Request**

**What would you like to be added**
Examine minikube pages to determine what should be hosted by minikube [docs](https://minikube.sigs.k8s.io/docs/) and what should be hosted by k8s docs.

Following the discussion on dual-hosted content (#15576), I think guides for installing minikube should be in the minikube project's docs, and then the k8s docs link to those installation guides.  State that the prerequisite for the Installing Kubernetes with Minikube page is successful installation of minikube and point to minkube docs. 
Already the k8s docs are stale with regard to container runtimes that minikube supports. 

k8s-specific how-to pages should remain in k8s docs.

Candidates to be removed from k8s docs and/or moved to minikube docs:

- [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) this page is similar to https://minikube.sigs.k8s.io/docs/start/linux/
- [Installing Kubernetes with Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/)
  - this whole page needs to be examined and reworked
  - [minikube features](https://kubernetes.io/docs/setup/learning-environment/minikube/#minikube-features) section duplicates (and omits) content in https://minikube.sigs.k8s.io/docs/overview/
  - [Starting a Cluster](https://kubernetes.io/docs/setup/learning-environment/minikube/#starting-a-cluster)  specifying the k8s version, specifying a VM driver, starting the cluster on alternate container runtimes, and use local images by re-using the Docker daemon sections are candidates to move to minikube docs (if not already there)
  - [Add-ons](https://kubernetes.io/docs/setup/learning-environment/minikube/#add-ons) section
  - [Additional Links](https://kubernetes.io/docs/setup/learning-environment/minikube/#additional-links) section
  - [Community](https://kubernetes.io/docs/setup/learning-environment/minikube/#community) section
  - [Hello Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/) references this page

**Why is this needed**
Dual-hosted content places undo burden on maintainers to ensure content is up-to-date in both places.

With all minikube installation configuration instructions on the minikube site, hopefully all minikube environment errors will be directed to the minikube team. 

**Comments**
Relates to #15576 Clarify style guide: eliminate third-party content
Relates to #15748 Umbrella Issue: Remove Third-Party Content
Also see closed issue #6339 [Restructure and Rewrite Minikube docs](https://github.com/kubernetes/website/issues/6339)

### LINKED ISSUE #22065: Document memory requirements for Minikube
**This is a Bug Report**

**Problem:**
Minikube installation never tell us how much memory it requires, in any OS flavor

**Proposed Solution:**
Please provide the base memory limits to start a minikube 

**Page to Update:**
https://kubernetes.io/...

### LINKED ISSUE #23650: Issue with k8s.io/docs/setup/learning-environment/minikube/
**This is a Bug Report**

**Problem:**
information on this page as well as on https://kubernetes.io/docs/tasks/tools/install-minikube/ is **out of date**.

**Proposed Solution:**
Remove /docs/tasks/tools/install-minikube/
Replace link destination from `docs/tasks/tools/install-minikube/` to `https://minikube.sigs.k8s.io/docs/start/`

**Page to Update:**
https://kubernetes.io/docs/setup/learning-environment/minikube/

### PR DESCRIPTION
**Pull request by sftim**
Replace existing [Learning Environment](https://kubernetes.io/docs/setup/learning-environment/) page with a redirect to [Install Tools](https://kubernetes.io/docs/tasks/tools/).

[Preview](https://deploy-preview-24533--kubernetes-io-master-staging.netlify.app/docs/setup/)

Accompanied by small related tweaks.

Fixes #15969
Fixes #22065
Fixed #23650
Relevant to issue #23354
Probably also fixes issue #24386

### CODE REVIEW

File: content/en/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -16,19 +16,20 @@ card:
 <!-- overview -->
 
 This tutorial shows you how to run a sample app
-on Kubernetes using [Minikube](/docs/setup/learning-environment/minikube) and Katacoda.
+on Kubernetes using [minikube](/docs/setup/learning-environment/minikube) and Katacoda.
```
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/tutorials/hello-minikube.md:19]: Is this
```suggestion
on Kubernetes using [`minikube`](/docs/setup/learning-environment/minikube) and Katacoda.
```
the change you're suggesting @irvifa?

[the remark this answers is not in the document]
Comment (irvifa) [reply content/en/docs/tutorials/hello-minikube.md:19]: It's more like question though since I'm not that sure how we plan to standardize this, if any

**Review by onlydole — APPROVED** [review]
Thank you, @sftim! This looks great to me! 

/lgtm

**Pull request merged**